sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -69,7 +71,7 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
                 # multiple seat types the one with the higher precedence will be added
                 # in course_type
                 if not sailthru_content_vars.get('course_type', None) or \
-                        seat_priority.index(sailthru_content_vars['course_type']) < seat_priority.index(seat['type']):
+                        seat_priority.index(sailthru_content_vars['course_type']) > seat_priority.index(seat['type']):
```

Comment:
If this is a bug fix, please create another PR and fix this bug so we can merge that one in quickly and run this script against Sailthru prod with that bug fix. I want this fix to be in quicker, not being buried with this PR.